presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -1047,6 +1047,38 @@ Invalidate Directory List Cache
 Invalidate Metastore Cache
 ^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+Clears all or part of the Hive Metastore cache on the coordinator.
+
+Full cache flush:
+
+.. code-block:: sql
+
+   CALL hive.system.invalidate_metastore_cache();
+
+Invalidate a schema:
+
+.. code-block:: sql
+
+   CALL hive.system.invalidate_metastore_cache(schema_name => 'web');
+
+Invalidate a table:
+
+.. code-block:: sql
+
+   CALL hive.system.invalidate_metastore_cache(
+       schema_name => 'web',
+       table_name  => 'clicks');
+
+Invalidate a single partition:
+
+.. code-block:: sql
+
+   CALL hive.system.invalidate_metastore_cache(
+       schema_name       => 'web',
+       table_name        => 'clicks',
+       partition_columns => ARRAY['ds'],
+       partition_values  => ARRAY['2025-09-18']);
+
```

Comment:
In my opinion, this full section seems to duplicate what’s already documented just below. If we’d like to include examples with sample values, we could simply add brief one-line examples alongside the existing documentation.